src/lightmap.cpp

```diff
@@ -447,7 +447,7 @@ void map::generate_lightmap( const int zlev )
 
     if( g->u.has_active_bionic( bionic_id( "bio_night" ) ) ) {
         for( const tripoint &p : points_in_rectangle( cache_start, cache_end ) ) {
-            if( rl_dist( p, g->u.pos() ) < 15 ) {
+            if( rl_dist( p, g->u.pos() ) < 2 ) {
```

Comment:
Why change this?

Reply:
Please read about Artificial Night Generator in PR's description

Reply:
Oh I missed your explanation sorry.

Looking at your math couldn't you make the power draw 852 kJ?

Reply:
We have only a little over 2000 kJ available. What would be the point of that bionic if it's unusable?

Reply:
Also power draw that high is unrealistic for any normal device.